contracts/cw721-non-transferable/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "cw721-non-transferable"
+authors = [
+  "Eliseo Cohen <eliseoci@users.noreply.github.com>",
+]
+version = "0.1.0"
```

Comment:
All other crates in this repo are at version `0.15.0`. Keep consistent?

Reply:
makes sense - changed it to `0.15.0`